### PR TITLE
Improve frontend

### PR DESCRIPTION
**Pull request by aquintanas**
- Add bootstrap
- Modify text appearing on mouse to fit EU format ( ,.2f€)
-  Modify text format in tables

### CODE REVIEW

**Comment by aquintanas** [comment]
Screenshot 2022-01-17 at 17 26 37

Screenshot 2022-01-17 at 17 21 54

Screenshot 2022-01-17 at 17 22 07

File: dashboard/dashboard.py [hunk 1]
```diff
@@ -15,8 +15,8 @@
     {
         'href': 'https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css',
         'rel': 'stylesheet',
-        'integrity': 'sha384-Uu6IeWbM+gzNVXJcM9XV3SohHtmWE+3VGi496jvgX1jyvDTXfdK+rfZc8C1Aehk5',
-        'crossorigin': 'anonymous'
+        #integrity': 'sha384-Uu6IeWbM+gzNVXJcM9XV3SohHtmWE+3VGi496jvgX1jyvDTXfdK+rfZc8C1Aehk5',
+        #'crossorigin': 'anonymous'
```
Comment (limdor) [inline dashboard/dashboard.py:19]: Shouldn't we keep it? And thinking about it, wouldn't be better to add it also to the other css?

**Review by limdor — COMMENTED** [review]
If I understood right the PR, we should be able to solve the 2 issues without the new bootstrap. Could you split it? I just tried locally and with the new bootstrap the visualization it got slightly worst for me. I'm using Chrome.

File: dashboard/dashboard.py (hunk 1 above)
Comment (aquintanas) [reply dashboard/dashboard.py:19]: If you check the console of the browser with this integrity check it raises a lot of errors. Indeed, bootstraps was not working before.

Comment (aquintanas) [reply dashboard/dashboard.py:19]: I fixed it. Thanks.

**Review by limdor — APPROVED** [review]

**Pull request merged**